src/ripple/core/impl/Stoppable.cpp

```diff
@@ -201,13 +195,6 @@ RootStoppable::stop(beast::Journal j)
     // Must have a prior call to start()
     assert(m_started);
 
-    if (stopAsync(j))
-        stopRecursive(j);
-}
```

Comment:
`stopAsync` returns true on the first time it is called, so this is just a guard to call `stopRecursive` the first time as well. I think it's clearer to just fold `stopAsync` into this method.